### DISCUSSION TITLE
Restricting Common User Access to Creating New Collections

### DISCUSSION THREAD

**Discussion by ghost** [jellyfin/jellyfin#10843] [General]
Hello, I have the following question: the option to add new items to collections is available for regular users. This leads regular users to try creating new collections, even though they cannot view them without root access. I would like the possibility to prevent regular users from creating new collections.

**Comment by Shadowghost** [comment]
The feature is already in master (https://github.com/jellyfin/jellyfin/pull/8887) and will be part of 10.9. It adds a new user permission for collection management.
